src/main/com/fulcrologic/rad/type_support/date_time.cljc

```diff
@@ -240,13 +241,14 @@
    [string? => inst?]
    (html-datetime-string->inst *current-zone-name* date-time-string))
   ([zone-name date-time-string]
-   [(? ::zone-name) string? => inst?]
+   [(? ::zone-name) string? => (? inst?)]
```

Comment:
Providing less is a breaking change...need to consider implications

Reply:
Similar to the others, as you've said. This one wasn't throwing, but it was returning nil on an exception. I'm not sure what the when-not accomplishes here (unless it were outside the try)...perhaps:

`(when-not (str/blank? date-time-string) ...)` on all of these?

Reply:
Sure that would be good.